.jshintignore

```diff
@@ -4,3 +4,5 @@ app/assets/javascripts/jstoolbar/**/*
 app/assets/javascripts/jquery_noconflict.js
 app/assets/javascripts/project/**/*
 app/assets/javascripts/lib/**/*
+frontend/tests/*
+frontend/tests/**/*
```

Comment:
It is - this should probably also be reverted. However, I do remember, that jshint complaints are a thing with the tests, so therefore we could leave it be.